test/test_package.cpp

```diff
@@ -1065,15 +1060,57 @@ TEST(DDPackageTest, CloseToIdentity) {
+    static constexpr std::size_t CT_MAT_TRANS_NBUCKET      = 4096U;
+    static constexpr std::size_t CT_MAT_CONJ_TRANS_NBUCKET = 4096U;
+
+    static constexpr std::size_t CT_MAT_MAT_MULT_NBUCKET        = 1U;
+    static constexpr std::size_t CT_MAT_VEC_MULT_NBUCKET        = 1U;
+    static constexpr std::size_t UT_VEC_NBUCKET                 = 1U;
+    static constexpr std::size_t UT_VEC_INITIAL_ALLOCATION_SIZE = 1U;
+    static constexpr std::size_t UT_MAT_INITIAL_ALLOCATION_SIZE = 1U;
+    static constexpr std::size_t CT_VEC_KRON_NBUCKET            = 1U;
+    static constexpr std::size_t CT_MAT_KRON_NBUCKET            = 1U;
+    static constexpr std::size_t CT_VEC_INNER_PROD_NBUCKET      = 1U;
+    static constexpr std::size_t STOCHASTIC_CACHE_OPS           = 1U;
+};
+
+using DensityMatrixPackageTest = dd::Package<DensityMatrixSimulatorDDPackageConfigTesting::UT_VEC_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::UT_VEC_INITIAL_ALLOCATION_SIZE,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::UT_MAT_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::UT_MAT_INITIAL_ALLOCATION_SIZE,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::CT_VEC_ADD_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::CT_MAT_ADD_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::CT_MAT_TRANS_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::CT_MAT_CONJ_TRANS_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::CT_MAT_VEC_MULT_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::CT_MAT_MAT_MULT_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::CT_VEC_KRON_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::CT_MAT_KRON_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::CT_VEC_INNER_PROD_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::CT_DM_NOISE_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::UT_DM_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::UT_DM_INITIAL_ALLOCATION_SIZE,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::CT_DM_DM_MULT_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::CT_DM_ADD_NBUCKET,
+                                             DensityMatrixSimulatorDDPackageConfigTesting::STOCHASTIC_CACHE_OPS>;
```

Comment:
Is this the same Package Config that is later being created again in the QFR?
I'd rather like to avoid that duplication.

If it's not the same as in the QFR, even worse. Including the DD Package Tests in a build requires to instantiate and compile the package class for an additional set of template arguments.. further increasing the compile time.

Reply:
The Package Config for the Stochastic and Deterministic Package is defined in the QFR, mainly since I need to access OpType for the Stochastic Simulator. For testing, however, I need Package Configs for both Packages in the dd_package. Not sure how to solve this.

Reply:
For the Deterministic Package it should be easy since everything that is needed, is known in the DDPackage. So there is no need to define this packege type three times across all repositories.
As for the Stochastic one it's not that easy but what you could do is to instantiate it here with the current `qc::OpCount` in the QFR so that both instantiations (the one here and the one in the QFR) create the same code as long as the number of operations does not change in the QFR. In case that happens, the number can be lazily upgraded here at a later point in time.

Reply:
While this avoids code duplication for the deterministic package definition, I am not sure this is worth it. We would lose the advantage of having all custom ddPackages for the simulators defined in the same place, i.e, `Simulator.hpp`.

At the end of the day I have not strong feelings about this, but I feel the current solution is "cuter".

Reply:
I see your point. Although I would try to create as few different packages as possible.
Thus, I would argue that the instances created in all the repositories should be identical.
As discussed above, this requires hard-coding the `OpCount` for the Stochatic Noise DD Package in the DDPackage repo, but that's a caveat I can live with.

Reply:
Done. All Density/Stoch Packages use the same configuration. As discussed, `OpCount` is hardcoded for the stochastic package in `test_package.cpp`